.github/workflows/lint.yml

```diff
@@ -31,7 +31,7 @@ jobs:
       - name: Install Dependencies
         run: pip install -r requirements/lint.txt
       - name: Black Format Check
-        run: black --check .
+        run: black --check . --exclude samples/
```

Comment:
I think this and the corresponding pre-commit-hook update is the right call. There will be other instances when we want to test multiline and similar things and we don't want black to format it if our integration tests are expecting non-formatted code.

Reply:
This change ended up not being needed for my PR but it still seems useful.